packages/calling/src/CallHistory/CallHistory.ts

```diff
@@ -56,12 +62,6 @@ export class CallHistory extends Eventing<CallHistoryEventTypes> implements ICal
 
   /**
    * Function to display the Janus API response.
-   *
-   * @param days - NUMBER_OF_DAYS.
-   * @param limit - LIMIT.
-   * @param sort - (ASC | DESC).
-   * @param sortBy - (startTime | endTime).
-   * @returns (user session response | error response).
```

Comment:
I think we still need to have the param names and descriptions or else users won't know what each parameter should have.